src/utils/stripe.ts

```diff
@@ -0,0 +1,16 @@
+import Stripe from "stripe";
+
+const stripeSecretKey = process.env.STRIPE_SECRET_KEY;
+
+if (!stripeSecretKey || stripeSecretKey.trim() === "") {
+  throw new Error("Stripe secret key is not set");
+}
+
+export const stripe = new Stripe(stripeSecretKey, {
```

Comment:
Create a function for stripe init? Including the checks above, accepting the `env` as argument?
Also a test for that would be nice (no matter how simple it may seem) 😄